--- FDCONFIG.SYS.orig	2017-05-03 10:40:51.830138998 -0500
+++ FDCONFIG.SYS	2017-05-03 10:43:16.861052664 -0500
@@ -15,7 +15,8 @@
 ; to learn more about menus and the FDCONFIG.SYS options.
 
 !MENUCOLOR=7,1
-!MENUDEFAULT=1,5
+; Load network by default
+!MENUDEFAULT=3,5
 
 MENU 1 - Load FreeDOS
 MENU 2 - Load FreeDOS without drivers
@@ -31,7 +32,8 @@
 
 !SET DOSDIR=C:\FDOS
 !COUNTRY=001,858,C:\FDOS\BIN\COUNTRY.SYS
-!LASTDRIVE=Z
+; we need lots of drives
+LASTDRIVE=Z
 !BUFFERS=20
 !FILES=40
 
@@ -48,8 +50,8 @@
 
 ; a) FreeDOS 1.0 Memory Manager (old, VirtualBox compatible) 
 
-1?DEVICE=C:\UTIL\EMM386\HIMEM.EXE
-1?DEVICE=C:\UTIL\EMM386\EMM386.EXE NOEMS X=EF00-EFFF
+; 1?DEVICE=C:\UTIL\EMM386\HIMEM.EXE
+; 1?DEVICE=C:\UTIL\EMM386\EMM386.EXE NOEMS X=EF00-EFFF
 
 ; b) JEMMEX (more features, might crash)
 
@@ -57,8 +59,9 @@
 
 ; c) JEMM386 and HIMEMX
 
-;1?DEVICE=C:\FDOS\BIN\HIMEMX.EXE
-;1?DEVICE=C:\FDOS\BIN\JEMM386.EXE X=TEST I=TEST I=B000-B7FF NOVME NOINVLPG
+; option a crashed with Invalid instruction sometimes
+1?DEVICE=C:\FDOS\BIN\HIMEMX.EXE
+1?DEVICE=C:\FDOS\BIN\JEMM386.EXE X=TEST I=TEST I=B000-B7FF NOVME NOINVLPG
 
 ; d) DR-DOS Memory Manager (trial-version, non-commercial use only).
 
@@ -98,5 +101,7 @@
 
 3456?SHELLHIGH=C:\FDOS\BIN\COMMAND.COM C:\FDOS\BIN /E:1024 /P=C:\AUTOEXEC.BAT
 
+; Reduce power usage at times, per http://help.fdos.org/en/hhstndrd/cnfigsys/idlehalt.htm
+IDLEHALT=-1
 
 
